tg_odesli_bot/platforms.py

```diff
@@ -73,7 +73,10 @@ class SpotifyPlatform(PlatformABC):
     """Spotify platform."""
 
     key = 'spotify'
-    url_re = r'https?://([a-zA-Z\d-]+\.)*spotify\.com/(album|track)/[^\s.,]*'
+    url_re = (
```

Comment:
New regex captures URLs like `https://spotify.com/family-plan/redeem/?token=VALUABLE_TOKEN` as well. I think we should avoid that since captured URLs along with (possible) private data are logged and sent to 3rd party.

Opening and closing parentheses can be omitted as well (`r'(https?... -> r'https?...` )

Reply:
all done :)

Reply:
Yep, looks fine now. How about splitting regex in concordance with its logical structure, like:
```
r'https?://([a-zA-Z\d-]+\.)*((spotify\.com/(album|track)/[^\s.,]*)'
r'|(tospotify\.com/[^\s.,]*))'```

Reply:
Ok, done